code-experiments/build/java/ExampleExperiment.java

```diff
@@ -182,6 +189,8 @@ public static void myRandomSearch(Function f,
 		    /* Call the evaluate function to evaluate x on the current problem (this is where all the COCO logging
 		     * is performed) */
 			y = f.evaluate(x);
+			if (numberOfConstraints > 0)
+				z = f.evaluateConstraint(x);
```

Comment:
Usually, I would expect the algorithm to call the constraint evaluation function before the function evaluation.